src/utils/release_activity_updater.rs

```diff
@@ -11,30 +11,45 @@ pub fn update_release_activity() -> Result<()> {
     let conn = try!(connect_db());
     let mut dates = Vec::new();
     let mut crate_counts = Vec::new();
+    let mut failure_counts = Vec::new();
 
-    for day in 1..31 {
+    for day in 0..30 {
```

Comment:
Did you mean to change the loop range here? It doesn't seem like the code farther down changed how it used it, so i'm wondering whether it was supposed to be like this originally.

Reply:
Yes, currently that loop is off by a day, this change is fixing it. I have been aware of this for a while but never had a chance to fix. This PR is a good excuse to fix it.